packages/hash/api/src/graphql/resolvers/knowledge/comment/replies.ts

```diff
@@ -0,0 +1,29 @@
+import { ApolloError } from "apollo-server-errors";
+import { CommentModel } from "../../../../model";
+import { ResolverFn } from "../../../apiTypes.gen";
+import { LoggedInGraphQLContext } from "../../../context";
+import {
+  UnresolvedPersistedCommentGQL,
+  mapCommentModelToGQL,
+} from "../model-mapping";
+
+export const persistedCommentReplies: ResolverFn<
+  Promise<UnresolvedPersistedCommentGQL[]>,
+  UnresolvedPersistedCommentGQL,
+  LoggedInGraphQLContext,
+  {}
+> = async ({ entityId }, _, { dataSources }) => {
+  const { graphApi } = dataSources;
+  const comment = await CommentModel.getCommentById(graphApi, { entityId });
```

Comment:
```suggestion
  const commentModel = await CommentModel.getCommentById(graphApi, { entityId });
```

Reply:
Renamed in dd02b703cafcbe2a034a4c0f67e1e12bbbdbe271